### PR TITLE
better info messages for activating new env

### PR DESCRIPTION
**Pull request by fredrikekre**

### CODE REVIEW

**Comment by Evizero** [comment]
much better

File: src/API.jl [hunk 1]
```diff
@@ -611,7 +611,7 @@ function activate(path::String; shared::Bool=false)
         # unless the shared environment already exists, place it in the first depots
         if !isdir(fullpath)
             fullpath = joinpath(Pkg.envdir(Pkg.depots1()), path)
-            @info("new shared environment \"$path\" will be placed at $fullpath")
+            @info("activating new shared environment \"$path\" at $fullpath.")
```
Comment (KristofferC) [inline src/API.jl]: Even though we use `--shared` I don't think we should call it `shared environment`. It's just a normal environment, no?

**Comment by KristofferC** [comment]
Looks like CI need to be fixed.

**Comment by Evizero** [comment]
Oh yea, I added tests for the log messages, which will need to be updated

**Pull request merged**